cluster/images/hyperkube/setup-files.sh

```diff
@@ -32,7 +32,7 @@ echo "admin,admin,admin" > /data/basic_auth.csv
 
 # Create HTTPS certificates
 groupadd -f -r kube-cert-test
-CERT_DIR=/data CERT_GROUP=kube-cert-test /make-ca-cert.sh $(hostname -i)
+CERT_DIR=/data CERT_GROUP=kube-cert-test /make-ca-cert.sh $(hostname -I | awk '{print $1}') IP:10.0.0.1,DNS:kubernetes,DNS:kubernetes.default,DNS:kubernetes.default.svc,DNS:kubernetes.default.svc.cluster.local
```

Comment:
This is nearly the same as #21379 
1) Use `hostname -I` for getting the real ip of the machine and only take the first IP address
Earlier, `hostname -i` often, if not always, returned `127.0.1.1` which doesn't say much.

2) This makes the api server communication a bit more "trusted", because we're signing it for `10.0.0.1` (internal ip) and the DNS default names. See also #18227

Will rebase after #21379 though